Makefile

```diff
@@ -52,7 +52,7 @@ release: certs docker-release
 	@echo "Built Docker image \"amazon/amazon-ecs-agent:latest\""
 
 gogenerate:
-	@cd agent && go generate ./...
+	@cd agent && GOOS=generate godep go generate ./...
```

Comment:
Not wild about using GOOS instead of go build -tags.  I think this is fine for now, but we're going to need to change this if we ever want multiple build constraints to be considered during compilation.